sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/dynamodb/DynamoDBIOTest.java

```diff
@@ -48,6 +48,7 @@
 import org.mockito.Mockito;
 
 /** Test Coverage for the IO. */
+@Ignore
```

Comment:
Maybe worth to add comment of the reason for the ignore, with something like `@Ignore("reason is ...")`

Reply:
Good point, updated now.